tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/EditVenue/DetailsTests.cs

```diff
@@ -0,0 +1,179 @@
+                telephone: "01234 567890",
+                website: "provider.com/test-venue",
+                addressLine1: "Test Venue line 1",
+                town: "Town",
+                postcode: "AB1 2DE");
+
+            var request = new HttpRequestMessage(HttpMethod.Get, $"venues/{venueId}");
+
+            // Act
+            var response = await HttpClient.SendAsync(request);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+
+            var doc = await response.GetDocument();
+            Assert.Equal("Test Venue", doc.GetSummaryListValueWithKey("Location name"));
+            Assert.Equal(
+                "Test Venue line 1\nTown\nAB1 2DE",
+                doc.GetSummaryListValueWithKey("Address"));
+            Assert.Equal("test-venue@provider.com", doc.GetSummaryListValueWithKey("Email"));
+            Assert.Equal("01234 567890", doc.GetSummaryListValueWithKey("Phone"));
+            Assert.Equal("provider.com/test-venue", doc.GetSummaryListValueWithKey("Website"));
+        }
+
+        [Fact]
+        public async Task Get_ValidRequestWithExistingFormFlowInstance_RendersExpectedOutputFromState()
+        {
+            // Arrange
+            var providerId = await TestData.CreateProvider();
+            var venueId = await TestData.CreateVenue(providerId);
+
+            var formFlowInstance = await CreateFormFlowInstance(venueId);
+            formFlowInstance.UpdateState(state =>
+            {
+                state.Name = "Updated name";
+                state.Email = "updated@provider.com";
+                state.PhoneNumber = "02345 678901";
+                state.Website = "updated-provider.com";
+                state.AddressLine1 = "Updated line 1";
+                state.AddressLine2 = "Updated line 2";
+                state.Town = "Updated town";
+                state.County = "Updated county";
+                state.Postcode = "UP1 D8D";
+                state.Latitude = 42;
+                state.Longitude = 42;
+                state.NewAddressIsOutsideOfEngland = false;
+            });
+
+            var request = new HttpRequestMessage(HttpMethod.Get, $"venues/{venueId}");
+
+            // Act
+            var response = await HttpClient.SendAsync(request);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+
+            var doc = await response.GetDocument();
+            Assert.Equal("Updated name", doc.GetSummaryListValueWithKey("Location name"));
+            Assert.Equal(
+                "Updated line 1\nUpdated line 2\nUpdated town\nUpdated county\nUP1 D8D",
+                doc.GetSummaryListValueWithKey("Address"));
+            Assert.Equal("updated@provider.com", doc.GetSummaryListValueWithKey("Email"));
+            Assert.Equal("02345 678901", doc.GetSummaryListValueWithKey("Phone"));
+            Assert.Equal("updated-provider.com", doc.GetSummaryListValueWithKey("Website"));
```

Comment:
I'd switch all the assertions to FluentAssertions as we've started to do elsewhere for better error messages and slightly nicer readability of test code.
I'd also wrap these blocks of assertions in an `AssertionScope` so that if there are problems you get all the errors at once. https://fluentassertions.com/introduction#assertion-scopes

Reply:
Done